### ISSUE TITLE
Adding smooth/feedback to the gyro mouse.

### ISSUE THREAD

**Issue by goathens** [kozec/sc-controller#566]
I'm very happy to see another project providing some functionality to windows steam controller users.

I'm running the windows pre-release.
I wanted to smooth out the gyro a bit, as is done with the touchpad.
I noticed that the gyro sensitivity, filter, settings cannot be changed in the GUI, so I edited the text file:

```
"gyro": {
    "action": "
        mode(RPADTOUCH, 
        smooth(8, 0.78, 5, 
        feedback(RIGHT, 28756, 100000000, 
        sens(2, -2, 
        mouse(YAW
    )))))"
},
```

However, when I look at the gui after reloading the config, it scrapes out the smooth and feedback settings:

````
"gyro": {"action": "mode(RPADTOUCH, sens(2, -2, mouse(YAW)))"}, 
````

Are smooth and feedback not compatible with the gyro?  Also I don't think feedback works on the touchpads at all in windows, but that's a separate issue.

**Comment by kozec** [comment]
> Are smooth and feedback not compatible with the gyro?

That's correct, at least currently. It shouldn't be problem to add smoothing to gyro input, but i'm not sure how feedback should work?

**Comment by LinuxJoeUser** [comment]
I'm gonna second and third smoothing for the gyro.  (I hadn't realized Patola had closed this in #258 (he erroneously thought setting smoothing for a pad would smooth the gyro if the gyro was acitvated via one of the pads.)

Please add smoothing to the original SCC as well as the Windows/C version. (*Big, sad, puppy-dog eyes*)
